Source/Moq.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net45;netstandard1.3</TargetFrameworks>
+		<TargetFrameworks>net45;netstandard2.0</TargetFrameworks>
```

Comment:
We first need to decide which frameworks we want to target before dropping older versions. This will likely need to be changed. I'll give you further feedback once we've figured out things.

Reply:
It would be great to support the latest netstandard2.0 version in order to use it at .net core 2.x projects and also the previous versions if it's possible.